### PR TITLE
5.x fix cue-points with a startTime of 0

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
Wait for tech to be ready before worrying about timeupdate events

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [x] If necessary, more likely in a feature request than a bug fix
  - [x] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [x] Unit Tests updated or fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/tracks/text-track.js [hunk 1]
```diff
@@ -331,7 +335,7 @@ class TextTrack extends Track {
   addCue(originalCue) {
     let cue = originalCue;
 
-    if (!(originalCue instanceof window.vttjs.VTTCue)) {
+    if (window.vttjs && !(originalCue instanceof window.vttjs.VTTCue)) {
```
Comment (gkatsev) [inline src/js/tracks/text-track.js]: I was mocking vttjs out https://github.com/videojs/video.js/pull/4146/files#diff-3b4ce30c38112a433d137fb9bf1bcbc9R21 but this probably is a better choice.

File: test/unit/tracks/text-track.test.js [hunk 2]
```diff
@@ -25,7 +27,7 @@ TrackBaseline(TextTrack, {
   mode: 'disabled',
   label: 'English',
   language: 'en',
-  tech: defaultTech
+  tech: new TechFaker()
```
Comment (gkatsev) [inline test/unit/tracks/text-track.test.js]: should this be `this.tech`?

Comment (BrandonOCasey) [inline test/unit/tracks/text-track.test.js]: its a suite of tests that runs on it's own, so it won't have access to `this.tech` as it is not in a `QUnit.test`

Comment (gkatsev) [inline test/unit/tracks/text-track.test.js]: oh, I see. LGTM then.

**Pull request merged**